website/content/docs/configuration/vault.mdx

```diff
@@ -186,6 +190,28 @@ will be removed in a future release.
 - `ttl` `(string: "")` - Specifies for how long the workload identity should be
   considered as valid before expiring.
 
+- `extra_claims` `(map[string]string: optional)` - A set of key-value pairs that
+  will be provided as extra identity claims for workloads. You can use the keys
+  as user claims in Vault role configurations. The values are interpolated. For
```

Comment:
Link `user claims in Vault role configurations` to https://developer.hashicorp.com/vault/api-docs/auth/jwt#user_claim